backend/endpoints/tests/test_identity.py

```diff
@@ -21,7 +21,7 @@ def test_login_logout(admin_user):
     assert response.status_code == 405
 
     basic_auth = base64.b64encode(
-        "test_admin:test_admin_password".encode("ascii")
+        b"test_admin:test_admin_password"
```

Comment:
are these equivalent?

Reply:
Default argument for `.encode()` is `"utf-8"`. However, as this testing string doesn't include any non-UTF8 characters, it is indeed equivalent.

```python
>>> b64encode("test_admin:test_admin_password".encode("ascii")).decode()
'dGVzdF9hZG1pbjp0ZXN0X2FkbWluX3Bhc3N3b3Jk'
>>> b64encode("test_admin:test_admin_password".encode()).decode()
'dGVzdF9hZG1pbjp0ZXN0X2FkbWluX3Bhc3N3b3Jk'
>>> b64encode(b"test_admin:test_admin_password").decode()
'dGVzdF9hZG1pbjp0ZXN0X2FkbWluX3Bhc3N3b3Jk'
```